data/json/npcs/missiondef.json

```diff
@@ -1843,5 +1843,16 @@
       "success_lie": "What?!  You're lying, I can tell!  Ugh, forget it!",
       "failure": "Oh no!  My poor puppy..."
     }
+  },
+  {
+    "id": "MISSION_REACH_REFUGEE_CENTER",
+    "type": "mission_definition",
+    "name": "Reach Refugee Center",
+    "goal": "MGOAL_GO_TO_TYPE",
+    "difficulty": 1,
+    "value": 50000,
```

Comment:
Good point, I had no idea, which point I should put there. So I chose the similar one to the similar missions.
As to your proposal, should this value be omitted?